api/python/ai/chronon/staging_query.py

```
@@ -31,7 +31,7 @@ def StagingQuery(
     setups: Optional[List[str]] = None,
     partition_column: Optional[str] = None,
     engine_type: Optional[EngineType] = None,
-    dependencies: Optional[List[Union[TableDependency, Dict]]] = None,
+    dependencies: List[Union[TableDependency, Dict]] = [],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)